ocaml/pci/bindings/ffi_bindings.ml

```diff
@@ -264,15 +264,15 @@ module Bindings (F : Cstubs.FOREIGN) = struct
 
   let pci_lookup_name_1_ary =
     foreign "pci_lookup_name"
-      (Pci_access.t @-> string @-> int @-> int @-> int @-> returning string_opt)
```

Comment:
There are a few more occurences of functions taking strings and returning strings. Are they safe, or we just don't use them?

Reply:
These are the only one that write in the buffer if I am correct, so the other should be fine as they are